pyannote/core/annotation.py

```diff
@@ -489,9 +495,9 @@ def get_tracks(self, segment):
         ----
         This will return an empty set if segment does not exist.
         """
-        return set(self._tracks.get(segment, {}))
+        return set(self._tracks.get(segment, {}).keys())
```

Comment:
Why  `.keys()`?

Reply:
Well, just to makes things more obvious. It's more of a personal taste, I always have to think twice when someone is iterating over a dict without explicitly calling `.keys()`.